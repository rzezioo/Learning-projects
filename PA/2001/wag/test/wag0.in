4
1000 10
3 10
5 3
250 8
