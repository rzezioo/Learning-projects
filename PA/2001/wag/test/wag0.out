3 10
5 3
250 8
1000 10
